garak/__main__.py

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env python3
+
```

Comment:
These additions in utility classes may not provide the utility intended. Happy to leave them for now.

Would you ever expect a user to execute:
```
./__main__.py -m garak --model_type huggingface --model_name gpt2 --probes dan.Dan_11_0
```

Reply:
No. OTOH cli.py isn't intended to invoked either. Let's remove them both.